_includes/navigation.html

```diff
@@ -37,6 +37,11 @@
                     <span class="label label-pill label-info">{{ site.data.future-episodes.size }}</span>
                 </a>
             </li>
+             <li class="nav-item">
+                <a class="nav-link" href="http://werocktech.com/awesome-tech-communities/">
```

Comment:
yep, makes sense 👍 

Will also add to other external links and add to this PR 😄